### PR TITLE
docs: info about appBadge

### PR DESCRIPTION
**Pull request by m1ga**
just a small info that you'll need to register for notifications if you want to use `appBadge` on iOS

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/iOS.yml [hunk 1]
```diff
@@ -1088,6 +1088,18 @@ properties:
...
     summary: Value of the badge for the application's springboard icon.
+    description: |
+        In order to change the app icon badge number you will need to register for notifications.
+        Normally this is done by using local notifications or a push service like FCM.
+        ``` js
+        Ti.App.iOS.registerUserNotificationSettings({
+          types: [
+                Ti.App.iOS.USER_NOTIFICATION_TYPE_ALERT,
+                Ti.App.iOS.USER_NOTIFICATION_TYPE_SOUND,
+                Ti.App.iOS.USER_NOTIFICATION_TYPE_BADGE
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/iOS.yml:1099]: Only this one is required, the others should be optional. Can you specify this?

File: apidoc/Titanium/UI/iOS/iOS.yml [hunk 2]
```diff
@@ -1088,6 +1088,18 @@ properties:
 
   - name: appBadge
     summary: Value of the badge for the application's springboard icon.
+    description: |
+        In order to change the app icon badge number you will need to register for notifications.
+        Normally this is done by using local notifications or a push service like FCM.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/iOS.yml:1093]: FCM may not be a common term for non-Firebase users. Can you rephrase to `[Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) (FCM)`?

File: apidoc/Titanium/UI/iOS/iOS.yml (hunk 1 above)
Comment (m1ga) [reply apidoc/Titanium/UI/iOS/iOS.yml:1097]: you are right! I've copied it from another example :)

File: apidoc/Titanium/UI/iOS/iOS.yml (hunk 2 above)
Comment (m1ga) [reply apidoc/Titanium/UI/iOS/iOS.yml]: :+1:

**Review by hansemannn — APPROVED** [review]
